packages/config/config/devices/0x0115/haseman_r4d4.json

```
@@ -0,0 +1,726 @@
+// Z-Wave.Me
+// Haseman R4D4
+{
+	"manufacturer": "Z-Wave.Me",
+	"manufacturerId": "0x0115",
+	"label": "Haseman R4D4",
+	"description": "4 x relay and 4 x universal dimmers",
+	"devices": [
+		{
+			"productType": "0xf222",
+			"productId": "0x2111"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Group 1",
+			"maxNodes": 8,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Group 2",
+			"maxNodes": 8
+		},
+		"3": {
+			"label": "Group 3",
+			"maxNodes": 8
+		},
+		"4": {
+			"label": "Group 4",
+			"maxNodes": 8
+		},
+		"5": {
+			"label": "Group 5",
+			"maxNodes": 8
+		},
+		"6": {
+			"label": "Group 6",
+			"maxNodes": 8
+		},
+		"7": {
+			"label": "Group 7",
+			"maxNodes": 8
+		},
+		"8": {
+			"label": "Group 8",
+			"maxNodes": 8
+		},
+		"9": {
+			"label": "Group 9",
+			"maxNodes": 8
+		}
+	},
+	"paramInformation": {
+		"64": {
+			"label": "Channel 1 power up memory",
+			"description": "When Power Up memory is enabled, the module will save the actual status of the associated channel output in case of power break",
+			"valueSize": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"minValue": 0,
+			"maxValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"65": {
+			"label": "Channel 2 power up memory",
+			"description": "When Power Up memory is enabled, the module will save the actual status of the associated channel output in case of power break",
+			"valueSize": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"minValue": 0,
+			"maxValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
```

Since this device has many repetitive config params, I'd only define each for the first channel and copy the info using `$import` and overwrite the label. For example
```suggestion
			"$import": "#paramInformation/64",
			"label": "Power Up Memory: Channel 1"
```
and similar.